.travis.yml

```diff
@@ -51,6 +51,13 @@ before_install:
     "${bootstrap_cmd[@]}"
     # List installed packages versions.
     $PYTHON -m pip list
+    if [ $TRAVIS_OS_NAME = osx ]
```

Comment:
Could you comment on why this was necessary in case sometime in the future it's no longer necessary?